docs/server/settings.rst

```diff
@@ -281,9 +281,13 @@ Translation environment configuration settings.
 
 .. setting:: POOTLE_TM_SERVER
 
-.. versionadded:: 2.7
```

Comment:
It says first versionadded 2.7 and then versionchanged 2.7.3 listing the changes.